lib/chef/resource/apt_repository.rb

```diff
@@ -164,6 +164,10 @@ class AptRepository < Chef::Resource
       property :key_proxy, [String, nil, FalseClass],
         description: "If set, a specified proxy is passed to GPG via `http-proxy=`."
 
+      property :signed_by, [String, true, false, nil],
+        description: "If a string, specify the file and/or fingerprint the repo is signed with. If true, set Signed-With to use the specified key",
+        default: nil
```

Comment:
Should this default to true, or conditionally default to true?

Reply:
Since 16.04 is the oldest version currently supported, I think this should be true by default.
https://docs.chef.io/platforms/#commercial-support-4